pytplot/store_data.py

```diff
@@ -99,9 +99,20 @@ def store_data(name, data=None, delete=False, newname=None):
                   np.nanmax(data_quants[base_data[0]].data.index)]
         df = base_data
         spec_bins = None
-    else:             
+    else:
         df = format_ydata(data['y'])            
         times = data['x']
+
+        # If given a list of datetime objects - note that the datetimes in the list should NOT be timezone aware
+        # (i.e., times should be in UTC) - convert times to seconds since epoch
+        if any(isinstance(t, datetime.datetime) for t in times):
+            for tt, time in enumerate(times):
+                times[tt] = (time-datetime.datetime(1970, 1, 1)).total_seconds()
+        # If given a list of datetime string, convert times to seconds since epoch
+        elif any(isinstance(t, str) for t in times):
+            for tt, time in enumerate(times):
+                times[tt] = [pytplot.tplot_utilities.str_to_int(time)]
```

Comment:
And here it looks like you are making a list of single element lists, so times ends up looking like:

[[1], [2], [3], [4]]

instead of 

[1, 2, 3, 4]

Reply:
I'll make sure to break it down, fixing this shortly.